src/crashlog.cpp

```diff
@@ -324,6 +324,25 @@ char *CrashLog::LogRecentNews(char *buffer, const char *last) const
 	return buffer;
 }
 
+/**
+ * Create a timestamped filename.
+ * @param filename      The begin where to write at.
+ * @param filename_last The last position in the buffer to write to.
+ * @param ext           The extension for the filename.
+ * @param with_dir      Wether to append directory in front of filename.
```

Comment:
s/wether/whether/
Maybe something like "Whether to suffix the file name with the directory"? Not sure whether "append ... in front" or "suffix" would be the prefered way to write it in English though.